api/users/serializers.py

```diff
@@ -128,6 +132,40 @@ def profile_image_url(self, user):
         size = self.context['request'].query_params.get('profile_image_size')
         return user.profile_image_url(size=size)
 
+    def _validate_user_json(self, value, json_schema):
```

Comment:
Not sure these should be instance methods--there's no usage of self. How about co-locating all the jsonschema validation functions in your new `api/users/schemas.py` module?